src/modules/navigation/Nav.jsx

```diff
@@ -41,6 +42,13 @@ export const Nav = () => {
         clickState={clickState}
       />
       <SharingsNavItem clickState={clickState} />
+      <NavItem
+        to="/trash"
+        icon={<Icon icon={TrashIcon} />}
+        label="trash"
+        rx={/\/trash(\/.*)?/}
+        clickState={clickState}
+      />
```

Comment:
we "recently" did the strict opposite of this commit. Did you knew ? Maybe this way is the right way to do it, maybe we missed something, I let you judge on that

Reply:
Previous work is here https://github.com/cozy/cozy-drive/pull/3185/commits/a4a1ab1c46637840f697608fadbc32c1c3868e2e and https://github.com/cozy/cozy-drive/pull/3185/commits/b621c452ab9becaa4b58838d3f7c46be7a5a44bd

It seems okay because there is still the trash for nextcloud drives inside nextcloud folders. So now we will have : 
- a central trash that concerns "our drive"
- trash inside nextcloud folders that concern nextcloud folders